build.psm1

```diff
@@ -618,7 +618,8 @@ Fix steps:
 
             # Make sure ExperimentalFeatures from modules in PSHome are added
             # https://github.com/PowerShell/PowerShell/issues/10550
-            @("PSDesiredStateConfiguration.InvokeDscResource") | ForEach-Object {
+            $ExperimentalFeaturesFromGalleryModulesInPSHome = @()
```

Comment:
The `PSDesiredStateConfiguration.InvokeDscResource` exp.feature is no longer there - that is intentional, but I don't want to get rid of this code because it might be useful in the future to other modules, so leaving just empty array here.